controllers/ebpf/agent_controller.go

```diff
@@ -53,8 +53,8 @@ const (
 	dedupeDefault                 = "firstCome"
 	envDedupeJustMark             = "DEDUPER_JUST_MARK"
 	dedupeJustMarkDefault         = "true"
-
-	envListSeparator = ","
+	envEnableFlowRTT              = "ENABLE_RTT"
```

Comment:
you should append this in the agent config

https://github.com/netobserv/network-observability-operator/blob/ac6138a017062ab39230a912fae00c84b168dfd9/controllers/ebpf/agent_controller.go#L155-L156

Reply:
Got it...adding